Resources/Prototypes/SS220/Entities/Objects/Consumable/Drinks/drinks.yml

```
@@ -17,7 +17,7 @@
 
 - type: entity
   parent: DrinkGlassBase
-  id: DrinkBerryСompote
+  id: DrinkBerryCompote
```

```suggestion
  id: DrinkBerryCompote
```
Здесь верни как было и делай миграцию на исправленную версию. Карты поломались этим изменением

---

Сделал revert изменения этого файла.

---

Отлично, тут ты поменял. Теперь используй migration.yml и создавай новый прототип с правильным названием у нас в файлике drinks.yml

---

Новый прототип с правильным названием, в файле миграции ссылка на новый прототип. 

_как же тяжело изменить 1 букву в слове компот(_